portfolio/src/main/webapp/chart-script.js

```diff
@@ -0,0 +1,108 @@
+google.charts.load('current', {'packages': ['corechart']});
+// google.charts.setOnLoadCallback(drawChart);
+
+const MIN_DATE = new Date('1/22/20');
+
+let data = {};
+let date = '';
+
+/**
+ * initializes data
+ */
+function init() {
+  update();
+  const countries = ['Singapore', 'China'];
+  const countriesParam = countries.join();
+  fetch('/covid-data?countries=' + countriesParam)
+      .then((response) => response.json())
+      .then((obj) => {
+        data = obj;
+        drawChart();
+      });
+}
+
+/** Redraws chart based on date selected. */
+function drawChart() {
+  const dataTable = new google.visualization.DataTable();
+  dataTable.addColumn('string', 'Country');
+  dataTable.addColumn('number', 'Cases');
+
+  for (const country in data) {
+    if (data.hasOwnProperty(country)) {
+      dataTable.addRow([country, data[country][date]]);
+    }
+  }
+
+  const options = {
+    'title': 'Covid Cases',
+    'width': 600,
+    'height': 300,
+    'legend': {position: 'none'},
+  };
+
+  const chart = new google.visualization.BarChart(
+      document.getElementById('chart-container'));
+  chart.draw(dataTable, options);
+}
+
+/**
+ * Throttles a function
+ * @param {function} func
+ * @param {number} limit
```

Comment:
It's unclear from the doc here if limit is meant to be milliseconds / seconds / minutes